src/cmdlib.sh

```diff
@@ -279,7 +281,9 @@ runcompose() {
         (echo "ERROR: overlay/ directory is no longer supported, use overlay.d/"
          echo "ERROR: https://github.com/coreos/coreos-assembler/pull/639") 1>&2
         exit 1
-    fi    
+    fi
+
+    rm -vf "${overrides_active_stamp}"
```

Comment:
could we get by somehow without having to do this? are we re-using the same file each time we build?

Reply:
It's messy because this code is entangled with the logic in `cmd-build` which only creates a new build directory *after* this.  We probably should move this stuff into a new `tmp/build` and `mv` that into the new builddir or so.  But it'd be a larger reworking.